pkg/cloudprovider/types.go

```diff
@@ -245,8 +246,7 @@ func (ofs Offerings) Available() Offerings {
 // Compatible returns the offerings based on the passed requirements
 func (ofs Offerings) Compatible(reqs scheduling.Requirements) Offerings {
```

Comment:
This looks the same as the Get() function now. Do we need both?

Reply:
This returns the first matched whereas compatible returns all matches, we could update the function names to better capture that behavior.

Reply:
We could also just take the result from compatible anywhere we use `Get`. The only reason we get the first result in `Get` is because we know there will only be a single offering that meets the constraints passed. There's definitely an argument to just drop `Get` in favor of using compatible.